### PR TITLE
Official Gradle Wrapper Validation GitHub Action

### PR DESCRIPTION
**Pull request by JLLeitschuh**
See: https://github.com/gradle/wrapper-validation-action

### CODE REVIEW

File: .github/workflows/gradle-wrapper-validation.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: "Validate Gradle Wrapper"
+on: [push]
```
Comment (ZacSweers) [inline .github/workflows/gradle-wrapper-validation.yml]: Without `pull_request`, this won't actually run for external contributors

Comment (JLLeitschuh) [reply .github/workflows/gradle-wrapper-validation.yml]: Are you sure this is accurate?

Comment (ZacSweers) [reply .github/workflows/gradle-wrapper-validation.yml]: Pretty sure. Push only applies to branches on the project, and your PR is not a branch on the project. We had to fix this in our autodispose project as well when enabling actions

Comment (JLLeitschuh) [reply .github/workflows/gradle-wrapper-validation.yml]: You were right. Should be fixed.
Thanks!

**Comment by 4u7** [comment]
Thank you!

**Comment by 4u7** [comment]
Looks like actions are disabled for our repo. Will check if we can enable it.

**Comment by JLLeitschuh** [comment]
😢 https://github.com/JetBrains/kotlin/commit/9b7e904390ce4eff2a96cc6b529e793fa4440b98

**Comment by 4u7** [comment]
@JLLeitschuh  Unfortunately we can't enable workflows for JetBrains organisation and without it enabled every push fails validation and sends tons of emails. Sorry!

**Comment by JLLeitschuh** [comment]
No worries. I ran into this with Spring (Pivotal) and Square already.

**Pull request merged**